packages/common-all/src/dnode.ts

```diff
@@ -1170,18 +1170,15 @@ export class NoteUtils {
    * @param notes: All notes in `engine.notes`, used to check the ancestors of `note`.
    * @returns The color, and whether this color was randomly generated or explicitly defined.
    */
-  static color(opts: {
-    fname: string;
-    vault?: DVault;
-    engine: DEngineClient;
-  }): {
+  static color(opts: { fname: string; vault?: DVault }): {
     color: string;
     type: "configured" | "generated";
   } {
-    const ancestors = NoteUtils.ancestors({ ...opts, includeSelf: true });
-    for (const note of ancestors) {
-      if (note.color) return { color: note.color, type: "configured" };
-    }
+    // JYTODO: Re-enable the ancestor color logic later
```

Comment:
just over communicating/confirming, were you planning on getting this wrapped up for this pr or in a separate pr? the enginev3 unification should not remove any existing feature from the desktop plugin

Reply:
Hadn't decided yet.  How important is this feature though? It seems quite arbitrary, and its implementation is not good from a performance point of view.

Reply:
Did some more analysis, and unfortunately it's going to be quite difficult to re-implement this ancestor coloring feature.  IMO we should just remove it; it's not worth complicating the parsing logic.  I think it's debatable whether this behavior is desired anyway, and it probably won't affect many users.